index.html

```diff
@@ -0,0 +1,103 @@
+<html>
+    <head>
+        <title> Intuit account</title>
+        <link rel="stylesheet" type="text/css" href="style.css">
+        <link href="/your-path-to-fontawesome/css/all.css" rel="stylesheet">
+        <link rel="stylesheet" href="path/to/font-awesome/css/font-awesome.min.css">
+        <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    </head>
+    <body>
+        <section class="sign-in-info">
+            <div class="container">
+            <div class="common-styles">
+            <div class="signin-account">
+                <p>Already have an Intuit account?</p>
+            </div>
+            
+            <div class="forgot-password">
+                <p><a href="https://accounts.intuit.com/signup.html?offering_id=Intuit.ifs.mint&namespace_id=50000026&redirect_url=https%3A%2F%2Fmint.intuit.com%2Foverview.event%3Futm_medium%3Ddirect%26cta%3Dhero_sign_up_free_ProspectWeb%26ivid%3Da9b8df23-0d05-490a-a527-662b045fd445%26adobe_mc%3DMCMID%253D43778899234622625232808106119344280814%257CMCORGID%253D969430F0543F253D0A4C98C6%252540AdobeOrg%257CTS%253D1600358737%26ivid%3Da9b8df23-0d05-490a-a527-662b045fd445#" target="_blank">I forgot my User ID or Password</a></p>
+            </div>
+        </div>
+        
+        <div class="signin-btn">
+         <button type="button" name="sign-in" value="Sign In"> <i class="material-icons">lock</i> Sign In</button>
+        </div>
+    </div> 
+        </section>
+
+        <section>
+            <div class="logo">
+                <img src="images/descarga1.png" >
+            </div>
+          <div class="logos">
+            <img src="images/descarga2.png">
+            <img src="images/descarga3.png">
+            <img src="images/descarga4.png">
+          </div>
+         
+        </section>
+
+        <section id="sign-up">
+                <div class="form-size">
+                    <div class="form-header">
+                        <div>
+                            <h1>Create an Intuit account</h1>
+                        </div>
+                        <div>
+                            <h3>One account for everything Intuit, including Mint.</h3>
+                        </div>
+                    <div>
+                        <h3> <a href="https://accounts-help.intuit.com/app/intuit/1995107" target="_blank">Learn more</a></h3>
+                    </div>
+                    
+                   
+                </div>
+                <div class="element-style">
+                    <div>
+                        <label>Email address</label>
+                        <input type="email" name="email">
+                    </div>
+
+                    <div>
+                        <label>Phone (recommended)</label>
+                        <input type="number" name="number">
```

Comment:
`<input type="tel">` is preferred for phone number field

![p](https://user-images.githubusercontent.com/26856491/93535779-9cf85e00-f93f-11ea-9f35-2b6783b35f97.png)